src/index.html

```diff
@@ -10,8 +10,175 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+  <body class="row">
```

Comment:
Avoid using presentational attributes in HTML. So if you need to add styles to a body you can make it directly.
But it's redundant for this task 

```
body {
  ...
}
```